node/src/Commands.ts

```diff
@@ -494,6 +494,300 @@ export function createSetBit(
     ]);
 }
 
+/**
+ * Represents a signed or unsigned argument encoding for the {@link BaseClient.bitfield|bitfield} or
+ * {@link BaseClient.bitfieldReadOnly|bitfieldReadOnly} commands.
+ */
+export interface BitEncoding {
+    /**
+     * Returns the encoding as a string argument to be used in the {@link BaseClient.bitfield|bitfield} or
+     * {@link BaseClient.bitfieldReadOnly|bitfieldReadOnly} commands.
+     *
+     * @returns The encoding as a string argument.
+     */
+    toArg(): string;
+}
+
+/**
+ * Represents a signed argument encoding.
+ */
+export class SignedEncoding implements BitEncoding {
+    private static readonly SIGNED_ENCODING_PREFIX = "i";
+    private readonly encoding: string;
+
+    /**
+     * Creates an instance of SignedEncoding.
+     *
+     * @param encodingLength - The bit size of the encoding. Must be less than 65 bits long.
+     */
+    constructor(encodingLength: number) {
+        this.encoding = `${SignedEncoding.SIGNED_ENCODING_PREFIX}${encodingLength.toString()}`;
+    }
+
+    public toArg(): string {
```

Comment:
I think this needs a simple javadoc since it's a public function. Same with other classes that implement this function.

Reply:
As far as I can tell the doc is auto-inherited since it is documented in the interface that this is implementing, same for the other places in this file. If I mouse over the function in my IDE it displays the interface doc.